### PR TITLE
Fix flaky test

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

File: frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ const checkAbsenceTypeSelectionButtonsExistance = async (
   arrivalTime = '08:15',
   departureTime = '16:00'
 ) => {
+  await page.reload()
```
Comment (kimifoud) [inline frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts:89]: 🤔

Comment (jaahon) [reply frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts:89]: Would a `page.goto` to the list page be more clear in intent? @kimifoud

Comment (kimifoud) [reply frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts:89]: Should the fixtures be created before the actual test and first navigation?

**Review by kimifoud — CHANGES_REQUESTED** [review]
I would still want to get rid of the redundant `page.reload()` call.

Maybe remove and extract the following code in `beforeEach`
```
async function pairDeviceAndLoadPage() {
  const mobileSignupUrl = await pairMobileDevice(
    employee.data.id!, // eslint-disable-line
    fixtures.daycareFixture.id
  )
  await page.goto(mobileSignupUrl)
}
```

and call it in each test after `createPlacement()`. Maybe as the first thing in `checkAbsenceTypeSelectionButtonsExistence`.

**Review by kimifoud — APPROVED** [review]

**Pull request merged**